### PR TITLE
ENG-1869 CR generation from Git reports the bundle type too

### PR DESCRIPTION
**Pull request by firegloves**
- bundle types to constant
- CR generation from Git reports the bundle type too

### CODE REVIEW

File: lib/de_bundle.js [hunk 1]
```diff
@@ -2,6 +2,11 @@ const _ = require('lodash');
 const base64Img = require('base64-img');
 const { EntandoDeBundle, Tag, Details } = require('./models');
 
+const bundletTypes = {
```
Comment (ffleandro) [inline lib/de_bundle.js:5]: typo

**Review by ffleandro — CHANGES_REQUESTED** [review]
Looks good to me besides the typo in the variable name

File: lib/de_bundle.js (hunk 1 above)
Comment (firegloves) [reply lib/de_bundle.js]: thx for the hint, updated

**Review by ffleandro — APPROVED** [review]
looks good to me

**Pull request merged**